apps/src/templates/PaneHeader.jsx

```
@@ -178,6 +178,9 @@ export const PaneButton = Radium(function (props) {
 
   function onKeyDownWrapper(event) {
     const {onClick} = props;
+    if (!onClick) {
```

As stated in the description, some of these components use dom events instead of an `onClick`. I noticed while testing keyboard navigability that an error was being thrown on these buttons (not just the new one). You can still interact with the buttons with the keyboard so I think it's okay to silence that error for now.